src/components/shared/DropdownMenu/index.jsx

```diff
@@ -0,0 +1,85 @@
+import propTypes from 'prop-types';
+import React from 'react';
+import {
+  SelectContainer,
+  StyledLabel,
+  StyledSelect,
+} from './dropdown-menu.styles';
+
+/**
+ * shared dropdown menu
+ *
+ * @param {JSX.Element} children the child component to wrap.
+ * @param {String} label menu label
+ * @param {String} name menu name
+ * @param {JSX.Element} labelIcon menu label icon
+ * @param {String} leftIcon icon inside the menu
+ * @param {Boolean} small small menu height
+ * @param {Boolean} smallBorderRadius small border radius
+ * @param {Boolean} dark dark background
+ *
+ * @return {JSX.Element}
+ */
+function DropdownMenu({
+  children,
+  label,
+  name,
+  labelIcon: LabelIcon,
+  leftIcon,
+  small,
+  smallBorderRadius,
+  dark,
+}) {
+  return (
+    <div>
+      {label && (
+        <StyledLabel htmlFor={name}>
+          {LabelIcon && LabelIcon}
+          <span>{label}</span>
+        </StyledLabel>
+      )}
+
+      <SelectContainer
+        leftIcon={leftIcon}
+        small={small}
+        smallBorderRadius={smallBorderRadius}
```

Comment:
Cant see why this is named this way 🤔 
```suggestion
        borderRadius={borderRadius}
```

Reply:
this is not a number, this is true/false value if the border raduis small or not

Reply:
Can't see why we would need this, please remove the prop and use the current style.

Reply:
this is a shared component sometimes the menu have small raduis and sometimes not

Reply:
can be adjusted via styled components. Just like the implementation of shared `<Button/>`

Reply:
yes i already done that its adjust it by styled component ??